pyomo/gdp/plugins/bigm.py

```diff
@@ -160,7 +162,8 @@ def __init__(self):
             Disjunction: self._warn_for_active_disjunction,
             Disjunct:    self._warn_for_active_disjunct,
             Block:       self._transform_block_on_disjunct,
-            LogicalConstraint: self._warn_for_active_logical_statement,
+            LogicalConstraint: False, # We transform these before we call
```

Comment:
It would be a bug, but it would be our bug. I can put the error back in, but if the transformation is coded correctly then it's unreachable code, which is why I took it out.

Reply:
Oh, but are you seeing leave it out entirely? Then it would at least trigger the no-handler-registered error if we messed something up?

Reply:
OK, this turned out to be a very good idea, as it already revealed a bug in `logical_to_linear`...